packages/ui/src/types/authenticator/stateMachine/authMachine.ts

```diff
@@ -9,7 +9,7 @@ import { AuthEvent } from './event';
 /**
  * Intefrace for `authMachine` machine interpreter
  */
-export type AuthInterpreter = Interpreter<AuthContext, any, AuthEvent>;
+export type AuthInterpreter = Interpreter<AuthContext, any, AuthEvent, any>;
```

Comment:
The new fourth argument is for `xstate`'s `TypeState` TS [feature](https://xstate.js.org/docs/guides/typescript.html#typestates), which we don't use yet. TypeScript was getting confused without us explicitly passing this, so adding `any`. Shouldn't affect any of our typing as we don't use this.